experimental/generation/generator/templates/enumArrayProperty.lg.lg

```diff
@@ -15,11 +15,11 @@ ${ValueTemplate()}
 
 # AskTemplate
 - ```
-# Ask${property}
-- \${askEnumArray('${property}')\}```
+${'#'} Ask${property}
+- \${askEnumArray('${property}')}```
 
 # ValueTemplate
 - ```
-# ${property}(values)
+${'#'} ${property}(values)
```

Comment:
>${'#'}  [](start = 0, length = 7)

can you \# ?

Reply:
No. That then passes through as \#

---
In reply to: [423898807](https://github.com/microsoft/BotBuilder-Samples/pull/2378#discussion_r423898807) [](ancestors = 423898807)